doc/development/rfc/legal_aspects_of_code_contributions.md

```diff
@@ -0,0 +1,82 @@
+providing code to be included in the GRASS repository understand that
+the code will be released under the GPL >=2 license, and that the
+person providing the code has the right to contribute the code.  In
+order to verify this, the committer must have a clear understanding of
+the license themselves. When committing 3rd party contributions, the
+committer should verify the understanding unless the committer is very
+comfortable that the contributor understands the license (for instance
+frequent contributors).
+
+If the contribution was developed on behalf of an employer (on work
+time, as part of a work project, etc) then it is important that an
+appropriate representative of the employer understand that the code
+will be contributed under the GPL license. The arrangement should be
+cleared with an authorized supervisor/manager, etc.
+
+The code should be developed by the contributor, or the code should be
+from a source which can be rightfully contributed such as from the
+public domain, or from an open source project under a compatible
+license.
+
+All unusual situations need to be discussed and/or documented.
+
+Committers should adhere to the following guidelines, and may be
+personally legally liable for improperly contributing code to the
+source repository:
+
+ * Make sure the contributor (and possibly employer) is aware of the
+  contribution terms.
+ * Code coming from a source other than the contributor (such as
+  adapted from another project) should be clearly marked as to the
+  original source, copyright holders, license terms and so forth. This
+  information can be in the file headers, but should also be added to
+  the project licensing file if not exactly matching normal project
+  licensing (grass/COPYRIGHT.txt).
+ * Existing copyright headers and license text should never be stripped
+  from a file. If a copyright holder wishes to give up copyright they
+  must do so in writing to the GRASS [PSC](https://trac.osgeo.org/grass/wiki/PSC) before copyright messages
+  are removed. If license terms are changed, it has to be by agreement
+  (written in email is ok) of the copyright holders.
+ * When substantial contributions are added to a file (such as
+  substantial patches) the author/contributor should be added to the
+  list of copyright holders for the file in the file header.
+ * If there is uncertainty about whether a change is proper to
+  contribute to the code base, please seek more information from the
+  Project Steering Committee, other GRASS developers or the OSGeo
+  foundation legal counsel.
+
+Questions regarding GRASS GIS should be directed to the
+GRASS Development Team at the following address:
+
+Internet:
+  http://grass.osgeo.org/home/contact-us/
+
+Postal address:
+
+```
+ GRASS Development Team
+ c/o Dr Markus Neteler
+ Fondazione E. Mach - Research and Innovation Centre
+ GIS and Remote Sensing Unit
+ 38010 S. Michele all'Adige
+ Italy
+ email: neteler AT osgeo.org
+```
```

Comment:
Drop postal address as no longer valid and not really needed anyway.

The rest may perhaps remain as is, otherwise we'd need an official amendment to the document.

Reply:
The link http://grass.osgeo.org/home/contact-us/ goes here: https://grass.osgeo.org/about/credits/ 

Shall we change that? Also, Markus still appears as chair in the credits page, shall I change that?

Reply:
> Shall we change that? Also, Markus still appears as chair in the credits page, shall I change that?

FYI: I have changed it on the web site and replaced myself with @veroandreo .